commands/inspect_test.go

```
@@ -29,7 +29,20 @@ func TestCmdInspectFormat(t *testing.T) {
 	assert.Equal(t, "\"none\"", actual)
 
 	actual, _ = runInspectCommand(t, []string{"--format", "{{prettyjson .Driver}}", "test-a"})
-	assert.Equal(t, "{\n    \"IPAddress\": \"\",\n    \"URL\": \"unix:///var/run/docker.sock\"\n}", actual)
+	type ExpectedDriver struct {
+		CaCertPath     string
+		IPAddress      string
+		MachineName    string
+		PrivateKeyPath string
+		SSHPort        int
+		SSHUser        string
+		SwarmDiscovery string
+		SwarmHost      string
+		SwarmMaster    bool
+		URL            string
+	}
+	expected, _ = json.MarshalIndent(&ExpectedDriver{MachineName: "test-a", URL: "unix:///var/run/docker.sock"}, "", "    ")
```

Why swallow the error?